Automattic/simplenote-macos
# Dropping WKWebView initialization from Simplenote's App Delegate

### Fix
- Drops **WKWebView** initialization from the AppDelegate
- Encapsulates Markdown Rendering in ... 🥁  ... **MarkdownViewController**
- Dropped **NoteEditorViewController.editorScrollView** since it wasn't even wired
- Since we're there, we're also dropping an IBOutlet (referencing a SplitView subview) from the AppDelegate).

@etoledom Sir, may I bug you yet again? (last time this week, promise!!!)
Ref. #458

### Test
1. Log into your account
2. Open any note
3. Enable Markdown  (Info button > Markdown Enabled)

- [ ] Verify that pressing over the Markdown Preview button (first one on the left, toolbar area) toggles Markdown Preview.
- [ ] Verify that switching themes with Markdown Preview onscreen looks great
- [ ] Verify that switching Notes, with Markdown Preview active, simply dismisses MD Preview and enables the editor
### Release
These changes do not require release notes.
